.github/ISSUE_TEMPLATE/question.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Question
+about: Ask a question about the tool.
+title: ''
+labels: 'question'
+assignees: ''
+
+---
+
+<!-- **IMPORTANT!**
+Please make sure to look for an answer to your question in our documentation and the documentation before asking a question here.
+
+If you have a general question regarding the module use Discord `nuxt` channel. Thanks!
```

Comment:
I couldn't find the channel whose name is `create-nuxt-app`, which is why I use `nuxt` channel.
Which channel should I point for?